### DISCUSSION TITLE
Project Definition

### DISCUSSION THREAD

**Discussion by chamodshehanka** [sliit-foss/fcsc-web#1] [General]
Home Page
1 contact banner - pop up
2 QnA
3 Notice - Latest - NavBar
4 upcoming all events
5 short into
6 community logo / partners

Pages
About & hierarchy Board, SM links
community - into, sm links, join
Event - Upcoming events , Main events
Notice (Need a backend)

**Comment by sithumonline** [comment]
CSS Framework https://tailwindcss.com/docs/utility-first#overview

**Comment by sithumonline** [comment]
![image](https://user-images.githubusercontent.com/24352487/122665383-7a43f000-d1c4-11eb-8fca-a918fda96469.png)

**Comment by chamodshehanka** [comment]
@sithumonline Once you have done, put the design here

**Comment by sithumonline** [comment]
Design : https://www.figma.com/file/fxbBAP62uOBPt5AyvvFWNZ/FCSC-Web?node-id=0%3A1
